### PR TITLE
Add initial tests

### PR DESCRIPTION
**Pull request by bnemanich**
## Checklist before submitting

- [x] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/master/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/master/doc/)?
- [x] Did you write any [tests](https://github.com/open-ce/open-ce/blob/master/tests/) to validate this change?  

## Description
Added initial tests for torchvision. This is part of the work done in open-ce/open-ce#145.

The version currently being stored in torchvision.__version__ is 0.7.0a0, which is not correct. Setting BUILD_VERSION will set the value correctly.

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/master/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/master/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

**Review by cdeepali — APPROVED** [review]

File: recipe/build.sh [hunk 1]
```diff
@@ -21,4 +21,5 @@ then
     export CUDA_INCLUDE_PATH="${PREFIX}/include,${CUDA_HOME}/include,${CONDA_PREFIX}/include"
 fi
 
+export BUILD_VERSION=${PKG_VERSION}
```
Comment (npanpaliya) [inline recipe/build.sh:24]: Is this change needed?

Comment (bnemanich) [reply recipe/build.sh:24]: Yeah, without it torchvision.__version__ shows 0.7.0a0, which isn't a real tag in GitHub (it also isn't the correct version number).
